packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/types/query/QueryPaginationBuilderTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.amazonaws.amplify.amplify_datastore.types.query
```

Comment:
These generally go _under_ the copy-right header.

Reply:
Updated